mteb/tasks/Classification/eng/FinancialPhrasebankClassification.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class FinancialPhrasebankClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="FinancialPhrasebankClassification",
+        description="Polar sentiment dataset of sentences from financial news, categorized by sentiment into positive, negative, or neutral.",
+        reference="https://arxiv.org/abs/1307.5336",
+        dataset={
+            "path": "financial_phrasebank",
+            "revision": "1484d06fe7af23030c7c977b12556108d1f67039",
+            "name": "sentences_allagree",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=None,
+        domains=["News"],
+        task_subtypes=None,
+        license="cc-by-nc-sa-3.0",
+        socioeconomic_status=None,
```

Comment:
My rough guess for financial news.
```suggestion
        socioeconomic_status="medium",
```